README.md

```diff
@@ -1,6 +1,7 @@
 ## HPE 3PAR Volume Plugin for Docker
 
 HPE Docker Volume Plugin is an open source project that provides persistent storage and features for your containerized applications using HPE 3PAR StoreServ Storage arrays.
+It supports both block and file (NFS shares) based features.
 
```

Comment:
@imran-ansari Can you please add issue #683 and #688 as a Limitations. And trouble shooting for 683.